ts/store/reducers/entities/services/servicesAllIds.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * A reducer to store all service id
+ * It only manages SUCCESS actions because all UI state properties (like
+ * loading/error)
+ * are managed by different global reducers.
+ */
+
+import { Action } from "../../../../actions/types";
+import { ServicesIdsArray } from "../../../../sagas/messages";
+import { SERVICE_LOAD_SUCCESS } from "../../../actions/constants";
+
+export type ServicesAllIdsState = ServicesIdsArray;
+
+export const INITIAL_STATE: ServicesAllIdsState = [];
+
+const reducer = (
+  state: ServicesAllIdsState = INITIAL_STATE,
+  action: Action
+): ServicesAllIdsState => {
+  switch (action.type) {
+    case SERVICE_LOAD_SUCCESS:
```

Comment:
in general, when there's a logic that modifies the state based on some event, it must be explained